src/features/vote/helpers/calculateTimeRemaining.ts

```diff
@@ -8,6 +8,12 @@ export const calculateTimeRemaining = () => {
   let text = "00:00";
   // format difference
   if (difference > 0) text = Duration.fromMillis(difference).toFormat("hh:mm");
+  // add h and m to time.
+  const split = text.split(":");
+  const ls = `${split[0]}h`;
+  const rs = `${split[1]}m`;
+
+  text = `${ls}:${rs}`;
```

Comment:
i think [this will work](https://moment.github.io/luxon/docs/manual/formatting.html#escaping)
`if (difference > 0) text = Duration.fromMillis(difference).toFormat("hh'h':mm'm'");`